types/html-parser/index.d.ts

```diff
@@ -0,0 +1,54 @@
+// Type definitions for html-parser 0.11
+// Project: https://www.npmjs.com/package/html-parser
+// Definitions by: Vladimir Grenaderov https://github.com/VladimirGrenaderov,
+//                 Max Boguslavskiy <https://github.com/maxbogus>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+export type Callback = (arg: any) => any;
+export type Token = '>' | '/>' | '?>';
+
+export interface CallbacksOption {
+    // * @param {Function} [callbacks.attribute] Takes the name of the attribute and its value
```

Comment:
jsdoc syntax is incorrect. Should be like:
```
/**
 *
 */

Reply:
Fixed.

Reply:
You're still missing an extra `*` on the first line:
```
/**  ...
```

Reply:
I decided to get rid of JSdoc and make arguments self-explanatory.